Home_Work_3/Task_1.py

```diff
@@ -0,0 +1,21 @@
+# Задача 16: Требуется вычислить, сколько раз встречается некоторое число X в массиве A.
+# Пользователь в первой строке вводит натуральное число N – количество элементов в массиве.
+# В последующих  строках записаны N целых чисел Ai (можно использовать псевдогенерацию). Последняя строка содержит число X.
+# *Пример:*
+# 5
+#     7 -2 3 5 1
+#     3
+#     -> 1
+
+n = int(input("Введите количество элементов массива n: "))
+some_list = list(range(1, n))
+print(some_list)
+
+x = int(input("Введите число x: "))
+sumX = 0
+
+for i in range(0, len(some_list)):
```

Comment:
У вас ведь уже есть параметр, содержащий длину списка. Рассчитывать ее заново не обязательно.
```suggestion
for i in range(n):
```
А вообще можно проходиться напрямую по элементам списка не вообще не используя функцию `range`.